.studio/golang

```diff
@@ -184,6 +184,11 @@ function go_update_component() {
 # Adding auto tab complete
 complete -F _component_auto_complete go_update_component
 
+document "kill_running_service" <<DOC
+  kill a running component. This can be used to restart a component
+
+  example kill_running_service ingest-service
+DOC
```

Comment:
Adding the `document` element to add this function to the hab studio describe function.